.github/workflows/check-tce-homebrew-formula-pr.yaml

```diff
@@ -0,0 +1,25 @@
+name: E2E Test - TCE Homebrew Formula
```

Comment:
For consistency, can we name this `Check -` instead of `E2E Test -`. We have been using the `Check` to signify a PR check-type test which might not be as exhaustive as a full-blown E2E test which might take longer. It's definitely ok for the Check to be exactly the same as the E2E, but it's just easier to identify what the purpose of the action is.

Reply:
Makes sense, let me change it 👍